tests/test_onnx_v2.py

```diff
@@ -207,6 +243,42 @@ def _get_models_to_test(export_models_list):
     return sorted(models_to_test)
 
 
+if is_tf_available():
+    from transformers import (  # T5Model,
+        AlbertModel,
+        BartModel,
+        BertModel,
+        DistilBertModel,
+        GPT2Model,
+        GPTNeoModel,
+        LayoutLMModel,
+        MBartModel,
+        RobertaModel,
+        XLMRobertaModel,
+    )
+
+    TENSORFLOW_EXPORT_DEFAULT_MODELS = {
+        ("ALBERT", "hf-internal-testing/tiny-albert", TFAlbertModel, AlbertConfig, AlbertOnnxConfig),
```

Comment:
Why do we need to include the model and config classes here?

Also, for consistency with the PyTorch tests, would it make sense to use lowercase names like `albert`?

Reply:
We don't and we should :) thank you!